src/js/api/bulk-order.js

```diff
@@ -535,7 +535,6 @@ ripe.Ripe.prototype._importBulkOrder = function(name, orders, options = {}) {
     const description = options.description === undefined ? null : options.description;
     const dataJ = {
         name: name,
-        brand: brand,

```

Comment:
isn't it already optional? Since `const brand` can be null

Reply:
But it is preferably to leave it as the line below:

```js
if (brand) dataJ.brand = brand;
```

Reply:
oops, I didn't see it :eyes: you are right

Reply:
I was the one making the change below :p But I missed this.